guangbochen/kontainer-engine-tke-driver
# Add driver RemoveLegacyServiceAccount interface

**Problem:**
need to implement driver RemoveLegacyServiceAccount interface.

**Solution:**
Add RemoveLegacyServiceAccount function to the RKE driver.

**Issues:**
N/A

**Validation Cases:**
1. ensure the service account is being cleaned up properly - that’s the `netes-default` service account in the `default` namespace
2. the `kontainer-engine` account exists in the `cattle-system` namespace.  
3. run through a cluster create just to make sure that another basic functionality is working. e.g deploy a workload `nginx` to the cluster, so you know cluster communication is working.
